engineapi/proxy_replica.go

```diff
@@ -4,8 +4,8 @@ import (
 	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 )
 
-func (p *Proxy) ReplicaAdd(e *longhorn.Engine, address string, restore bool) (err error) {
-	return p.grpcClient.ReplicaAdd(p.DirectToURL(e), address, restore, e.Spec.VolumeSize, e.Status.CurrentSize)
+func (p *Proxy) ReplicaAdd(e *longhorn.Engine, address string, restore, fastSync bool) (err error) {
+	return p.grpcClient.ReplicaAdd(p.DirectToURL(e), address, restore, e.Spec.VolumeSize, e.Status.CurrentSize, fastSync)
```

Comment:
note: For backward compatibility, when longhorn-manager is 1.4.0 but instance-manager-engine is 1.3.x, it should still communicate with the old implementation because the proto request payload is compatible.